rte/hack/manifests/sample-devices/daemonset-dpa.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  namespace: sampledevices
+  name: dpa-ds
+spec:
+  selector:
+    matchLabels:
+      name: dpa
+  template:
+    metadata:
+      labels:
+        name: dpa
+    spec:
+      serviceAccountName: sampledevices-sa
+      containers:
+        - name: dpa-container
+          securityContext:
+            privileged: true
+          image: quay.io/k8stopologyawareschedwg/sample-device-plugin:v0.1.1
```

Comment:
This image is fine for past and future releases?
If not we should make it configurable